source/options.html

```diff
@@ -26,6 +26,14 @@ <h3>Color Previewer</h3>
 		</div>
 		<div class="color-output"></div>
 	</div>
+	<h3>Warning Content</h3>
+	<div class="text-inputs">
+			<label class="text-input">
+				<span>Text</span>
+				<input type="text" name="text">
+			</label>
+		<div class="text-output"></div>
```

Comment:
What's happening here, is this element being used? I don't see any changes in `options.js` related to it.

Reply:
Do you mean the text-output? If so, not using indeed, and kept it by mistake when duplicating the element.